sched/scheduler/jobStatus.go

```diff
@@ -0,0 +1,83 @@
+package scheduler
+
+import (
+	s "github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched"
+)
+
+type JobStatus struct {
+	Id         string
+	Status     sched.Status
```

Comment:
I think we already do this elsewhere. I can't think of anything better unless you'd prefer JobStatus and JobStatusMsg.